git_jira/cli.py

```diff
@@ -1,28 +1,41 @@
-from typing import List, Optional
-
 import click
-# import yaml
-
-# def get_yaml_content(file_path: str) -> dict:
-#     with open(file_path) as f:
-#         return yaml.safe_load(f)
-
-# CONFIG = get_yaml_content("config.yaml")
+from git_jira.config import CONFIG_FILE_PATH, config_params_to_str, write_config_file, config_file_exists
 
 @click.group()
 def cli():
     pass
 
+@click.command()
+def config():
+    click.echo("Welcome to git-jira. Let's generate your config file...")
+    jira_server_url = click.prompt("Insert your Jira server url", type=str)
+    jira_username = click.prompt("Insert your Jira username", type=str)
+    jira_api_token = click.prompt("Create an API token here https://support.atlassian.com/atlassian-account/docs/manage-api-tokens-for-your-atlassian-account/ \nand insert your Jira API token", type=str)
+    jira_default_project_key = click.prompt("Select default project to use. \nLeave empty if you want to specify it as flags. Insert the short code (like TP, PJ, etc)", type=str) 
+    click.echo(f"A config.yaml file with the following params will be created at {CONFIG_FILE_PATH}\n\n{config_params_to_str(jira_server_url, jira_username, jira_api_token, jira_default_project_key)}\n")
```

Comment:
If you create a dictionary with these keys, you can then use `yaml.dump(configs)` and get the yaml output.